modules/Quiz/resources/quiz.js

```diff
@@ -689,7 +689,7 @@
         isPortrait: function () {
             if (
                 this.getPlayer().getInterface().hasClass("mobile")
-                && (screen.orientation.angle === 0 || screen.orientation.angle === 180)
```

Comment:
@ErezBouskila  
This looks OK but I am afraid that at some point one of the element in the chain parent.document.body.parentNode.clientWidth or parent.document.body.parentNode.clientHeight will be undefined. Perhaps in some embed types like iframe) 
Can we add a safer test ( if parent && parent.document .... )